src/smpp_gateway/client.py

```diff
@@ -59,29 +63,37 @@ class PgSmppClient(smpplib.client.Client):
     https://gist.github.com/pkese/2790749
     """
 
-    def __init__(self, *args, **kwargs):
-        self.backend = kwargs.pop("backend")
-        self.submit_sm_params = kwargs.pop("submit_sm_params")
+    def __init__(
+        self,
+        notify_mo_channel: str,
+        backend: Backend,
+        submit_sm_params: Dict,
+        *args,
+        **kwargs,
+    ):
+        self.notify_mo_channel = notify_mo_channel
+        self.backend = backend
+        self.submit_sm_params = submit_sm_params
         super().__init__(*args, **kwargs)
         self._pg_conn = pg_listen(self.backend.name)
 
     # ############### Handlers ################
 
```

Comment:
I rearranged these methods to be "chronologically" ordered as you read through.  I was getting lost reading them from low-level to high-level.  I'm happy to get pushback on that, though.

Unfortunately it makes the diff the messy, but I will call out the (basically zero) changes I made.